CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ endif()
 INCLUDE(ExternalProject)
 INCLUDE(CheckCXXSourceCompiles)
 
-SET(CMAKE_CXX_STANDARD 11)
+SET(CMAKE_CXX_STANDARD 14)
```

Comment:
We use c++20 if compiler handles it [via compiler flags](https://github.com/mariadb-corporation/mariadb-columnstore-engine/blob/develop/CMakeLists.txt#L53C1-L53C1). Is this set is necessary?

Reply:
It is necessary. If it is not specified as 14 during compilation, an error will be reported.